resources/api-docs/1.0/entities.md

```diff
@@ -215,6 +216,25 @@ You can post an array with the ids of the entities you want to transform to the
 
 > {success} Code 200 with JSON.
 
+<a name="transfer-entities"></a>
+## Transfer Entities
+
+You can post an array with the ids of the entities you want to transfer to another campaign to the `/transfer` endpoint to transfer or copy them.
+
+| Method | URI | Headers |
+| :- |   :-   |  :-  |
+| POST | `transfer` | Default |
+
+| Parameter | Type | Description
+| :- | :- | :- |
+| `entities` | `array`(required) | The ids of the entities to transfer or copy. |
+| `campaign_id` | `integer`(required) | The id of the campaign the entity will be transfered or copied to. |
+| `copy` | `boolean` | True if the entity will be copied, false if the entity will be transfered, defaults to copy if left empty |
```

Comment:
This is really confusing. The endpoint is called "transfer", but if you don't provide a copy param, it will copy? I know I said to copy the original way ti works, but this makes no sense. Copy should be optional and only copy if set to true. Also missing the end `.` to keep it consistent.